scp.js

```diff
@@ -32,12 +41,14 @@ scp.send = function (options, cb) {
  * Grab a file from a remote host
  */
 scp.get = function (options, cb) {
+  var ssh_options = ["ControlMaster=no"]; //callback is not fired if ssh sessions are shared
+  ssh_options = options.ssh ? ssh_options.concat(options.ssh) : ssh_options;
```

Comment:
can you make `ssh_options` camelCase please? nothing else in this project is snake_case